kratos/sources/kratos_version.cpp

```diff
@@ -59,6 +59,45 @@ KRATOS_SHA1_NUMBER "-" \
 KRATOS_BUILD_TYPE  "-" \
 KRATOS_ARCH_TYPE
 
+// Define OS name
+#if defined(__linux__) || defined(__linux) || defined(linux) || defined(__gnu_linux__)
+    #define KRATOS_OS_NAME "GNU/Linux"
+#elif defined(__APPLE__) && defined(__MACH__)
+    #define KRATOS_OS_NAME "Mac OS"
```

Comment:
I think you have a copy paste error here :imp:

Reply:
If I use the macros we define for some reason doesn't work, and with the copy paste it works....

Reply:
Oh no don't worry, its because of Mac :P